cli/tests/testdata/test/no_check.out

```diff
@@ -3,10 +3,10 @@ Uncaught error from ./test/no_check.ts FAILED
  ERRORS 
 
 ./test/no_check.ts (uncaught error)
-error: TypeError: Cannot read properties of undefined (reading 'fn')
+error: (in promise) TypeError: Cannot read properties of undefined (reading 'fn')
 Deno.test();
      ^
-    at [WILDCARD]/test/no_check.ts:1:6
+    at [WILDCARD]
```

Comment:
I think the file check here was okay. Is there a problem leaving it in?

Reply:
Oops. That was accidentally nuked. Will restore.